public/less/tweaks.less

```diff
@@ -24,3 +33,46 @@ select.form-control.styled-select {
 select.form-control.styled-select::-ms-expand {
     display: none;
 }
+
+.list-result{
+    .result-title{
+        font-size:14pt;
+    }
+    .right-badge{
+        background-color: @brand-primary;
```

Comment:
Isn't this pretty much what `label label-primary` would do, on in a more bootstrap standard way?